content/docs/user/artifact_gen/index.md

```diff
@@ -160,24 +160,23 @@ We only support sbt 1+, not sbt 0.13.
 {{< /alert >}}
 
 {{< alert warning >}}
-If you use the 'it' config, you have to use a custom build command as the defauit one is for the 'test' config: `sbt -J-Xss100M ;clean;it:assembly -batch --error`
+If you use the 'it' config, you have to use a custom build command as the default one is for the 'test' config: `sbt ;clean;it:assembly`
```

Comment:
I just realized: we can probably just get rid of this paragraph, since the one place where we explain which command you need to run already mentions both `test` and `it` configs? Plus it's located a bit further bellow, and this warning doesn't make much sense if you read it first... 🤔

Reply:
You have no idea how many people misunderstand "Gatling = Scala = I have to use sbt even if I don't know shit about it". Also, we have the situation where someone with Scala experience built the Gatling test suite and years later, the test suite is maintained by someone else who doesn't know sbt and this person is trying to evaluate FrontLine.

One way or another, we must provide the commands explicitly, and not expect users to figure out which configuration they have to use.

Reply:
We do provide both commands further below (line 192), but worded differently ("if you've configured the plugin for integration tests" rather than "If you use the 'it' config").

I think what I don't like here is that the text (outside the alert boxes) follows three clear steps (1. set up the build definition, 2. set up the build plugins, 3. run the build command), but the alert boxes are just bunched up in between steps 1 and 2 regardless of which step of the process they should help with.

Here's how I would structure it:

```
intro paragraph
[SBT 1+ warning]
[Coursier warning]

paragraph about build.sbt
[tip about gatling-test-framework being optional]

paragraph about project/plugins.sbt
[tip about gatling-sbt being optional]

paragraph about running the correct command
[rewritten warning explaining in which case you have to care about the it config - i.e., you're using gatling-sbt AND your Gatling tests are in src/it]
```

Reply:
I just pushed something like that in a separate commit; what do you think? If it's not an improvement I can just revert the last commit and only merge the rest.

Reply:
Way better, thanks!